### DISCUSSION TITLE
so-elastic and so-stenos missing

### DISCUSSION THREAD

**Discussion by Flizzah** [Security-Onion-Solutions/securityonion#11933] [Unsupported Versions] [answered]
Hello everyone, I installed SecurityOnion from an Azure image, and I believe the installation went well. However, after checking the services, I found that there are 4 services that are missing. 

![status](https://github.com/Security-Onion-Solutions/securityonion/assets/152970968/fe6fbf8f-2824-43ae-88e8-ac7c68e02d4b)

I did some research and tried some commands, but none of them worked. When attempting to restart the service, it shows "connection attempt timed out," prompting us to try a series of commands.

![After restart service](https://github.com/Security-Onion-Solutions/securityonion/assets/152970968/44100691-3c89-4e7f-817e-48e6631de93e)

 I also noticed that the log/elasticsearch directory is missing. 

Can someone help me with this issue? 
Thank you.

**Comment by Flizzah** [comment]
I run "sudo so-pcap-restart" and so-steno now is running

**Comment by dougburks** [comment]
Did you use our official Azure image at https://securityonion.net/azure?

Did you follow our documentation at https://docs.securityonion.net/en/2.4/cloud-azure.html?

How much CPU, RAM, and storage did you assign to the node?

**Reply by Flizzah** [reply]
The installation was done directly on Azure. During the setup, due to resource constraints, I chose 'Standard D2s v3 (2 vcpus, 8Gib memory). I found the resource requirements quite high based on the manual, and in the context of testing, I didn't think it would be an issue for now

**Reply by dougburks** [reply, marked as answer]
8GB RAM is not enough for a Standalone installation. The only installation you can do in 8GB RAM is an Import installation:
https://docs.securityonion.net/en/2.4/hardware.html#minimum-specs
